lib/services/workflow-api-service.js

```diff
@@ -155,7 +236,22 @@ function workflowApiServiceFactory(
     };
 
     WorkflowApiService.prototype.findActiveGraphForTarget = function(target) {
-        return taskGraphStore.findActiveGraphForTarget(target);
+        return waterline.graphobjects.findOne({
+            node: target,
+            _status: Constants.Task.ActiveStates
+        });
+    };
+
+    WorkflowApiService.prototype.getActiveWorkflows = function(query) {
+        return waterline.graphobjects.find(query);
+    };
+
+    WorkflowApiService.prototype.getWorkflowById = function(id) {
+        return waterline.graphobjects.needByIdentifier(id);
+    };
+
+    WorkflowApiService.prototype.destroyGraphDefinition = function(injectableName) {
```

Comment:
How above to use the uniform words for the same action of task definition and graph definition?
This is your current mapping:
```
destroyGraphDefinition <--> deleteWorkflowsTasksByName 
getGraphDefinitions  <--> getWorkflowsTasksByName 
defineTaskGraph <--> putWorkflowsTasksByName
```
If we use `destroy` for graph, then it should be better to use `destroy` for task as well; If we add `byName` for task, then it should also be better to add the words for graph.